demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -116,7 +115,7 @@ def is_valid_scheme(self):
         if self.scheme_name in [None, FileType.IMAGE, FileType.README, FileType.RELEASE_NOTES, FileType.TEST_PLAYBOOK]:
             return True
         # ignore reputations.json
-        if checked_type(self.file_path, JSON_ALL_REPUTATIONS_INDICATOR_TYPES_REGEXES):
+        if checked_type_by_reg(self.file_path, JSON_ALL_REPUTATIONS_INDICATOR_TYPES_REGEXES):
```

Comment:
if the entire purpose is to ignore `reputations.json` and avoid regex as much as possible maybe we can change this  to something like:
```
if self.scheme_name in FileType.REPUTATION and os.path.basename(self.file_path) == 'reputations.json'
```
Where 'reputations.json` should be put in a constant in case we would like to change this file name in the future.

Reply:
you are right